check_zookeeper.pl

```diff
@@ -277,7 +277,7 @@ BEGIN
             next;
         }
     }
```

Comment:
For temporarly solution, I've added exp to float converter:

`
# Convert exp to float
$mntr{$_} = sprintf("%.1f", $mntr{$_});
`